Req8-CreateView.sql

```diff
@@ -0,0 +1,10 @@
+CREATE VIEW viewEpisodes AS
+SELECT e.EpisodeId, e.SeriesNumber, e.EpisodeNumber, e.EpisodeType, e.Title, e.EpisodeDate,
```

Comment:
you can here list all columns by writing e.*, it's better and more dynamically if you add any new column into episode table